Added APP_FULLBASEURL

HarderWork opened the pull request:
Think this is helpful for new users, took me a while to figure it out.

HarderWork committed:
Added APP_FULLBASEURL

Think this is helpful for new users, took me a while to figure it out.

josegonzalez commented:
I don't want to merge this as is - maybe commented out is fine - since not everyone uses a vm or mounts the `.dev` prefix to localhost.

I personally use the internal CakePHP server, which this would break entirely.

HarderWork committed:
Commented out setting

Change to example.com to make it easier for devs to understand what it's used for.

josegonzalez commented on config/.env.default:
```diff
@@ -1,7 +1,7 @@
 
 export APP_NAME="__APP_NAME__"
-export APP_FULLBASEURL="http://${APP_NAME}.dev"
+#export APP_FULLBASEURL="http://example.com"
```
You can keep it with the appname thing.

HarderWork committed:
Using APP_NAME variable in url
